src/br/com/zup/nossacasacodigo/author/Author.java

```diff
@@ -13,35 +13,19 @@
 	private LocalDateTime createdAt;
 	
 	public Author(String name, String email, String description) {		
-		emailValidator(email);
-		nameValidator(name);
-		descriptionValidator(description);
-
-		this.name = name;
-		this.email = email;
-		this.description = description;
+		this.name = IsEmpty.check(name, "nome");
+		this.email = emailValidator(email);
+		this.description = descriptionValidator(description);;
 		this.createdAt = LocalDateTime.now();
 	}
 
 	//Getters
-	public String getEmail() {
+	public String getKey() {
```

Comment:
Esse método chama getKey muito provavelmente por conta da outra classe que usa o mapa, não é? Se isso for verdade, tenta fazer o seguinte exercício: se esse método fosse um mundo e você não conhecesse além dele, o nome dele seria getKey()?

Reply:
Havia mudado o nome para getKey pensando em uma das soluções para a abstração do método de inserir um elemento a database. Porém no momento atual acredito fazer mais sentido voltar para os nomes padrões de antes, como getEmail()